prepare_stmt.go

```diff
@@ -44,6 +44,12 @@ func (db *PreparedStmtDB) Close() {
 	}
 }
 
+func (db *PreparedStmtDB) Reset() {
+	db.Close()
```

Comment:
Maybe we need to range `db.Stmts` and close all prepared statement, `Close` will only close prepared statements in this session.